src/test/compile-fail/macro-missing-delimiters.rs

```diff
@@ -0,0 +1,17 @@
+// Copyright 2016 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+macro_rules! baz(
+    baz => () //~ ERROR invalid macro matcher;
```

Comment:
Hmm.. shouldn't it spawn a help message?

I guess it is not important enough to block landing this PR, but I am surprised